torchvision/models/efficientnet.py

```diff
@@ -32,15 +32,23 @@
 
 
 class SqueezeExcitation(nn.Module):
-    def __init__(self, input_channels: int, squeeze_channels: int):
+    def __init__(
+        self,
+        input_channels: int,
+        squeeze_channels: int,
+        activation: Optional[Callable[..., nn.Module]] = None,
+    ) -> None:
         super().__init__()
         self.fc1 = nn.Conv2d(input_channels, squeeze_channels, 1)
         self.fc2 = nn.Conv2d(squeeze_channels, input_channels, 1)
+        if activation is None:
+            activation = nn.SiLU
+        self.activation = activation(inplace=True)
```

Comment:
I would probably drop `inplace` from here because not all activation methods in PyTorch support the attribute (for example `Sigmoid`).